kalite/settings.py

```diff
@@ -169,7 +169,9 @@ def package_selected(package_name):
 
 import_installed_app_settings(INSTALLED_APPS, globals())
 
-KHAN_EXERCISES_DIRPATH = getattr(local_settings, "KHAN_EXERCISES_DIRPATH", os.path.join(os.path.dirname(__file__), "static", "perseus", "ke"))
+KHAN_EXERCISES_RELPATH = os.path.join("static", "perseus", "ke")
```

Comment:
This seems to be contra-indicated by Django convention:

from django.conf import settings

settings.DEBUG = True   # Don't do this!